scripts/plotting/polar_map.py

```diff
@@ -0,0 +1,381 @@
+from pathlib import Path  # python standard library
+from typing import Optional  # this is just for type hints
+
+# data loading / analysis
+import xarray as xr
+import numpy as np
+
+# plotting 
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import matplotlib.gridspec as gridspec
+from mpl_toolkits.axes_grid1.inset_locator import inset_axes
+import cartopy.crs as ccrs
+import cartopy.feature
+from cartopy.util import add_cyclic_point
+
+# ADF library
+import plotting_functions as pf
+
+
+def polar_map(adfobj):
+    """
+    This script/function is generates polar maps of model fields with continental overlays.
```

Comment:
Typo here (`function is generates` -> `function generates`)

Reply:
Done.